kratos/includes/gid_io.h

```diff
@@ -198,6 +198,9 @@ class GidIO : public IO
         mGidMeshContainers.push_back( TMeshContainer(
                                           GeometryData::Kratos_Point3D,
                                           GiD_Point, "Kratos_Point3D_Mesh" ) );
+        mGidMeshContainers.push_back( TMeshContainer(
```

Comment:
Can we put the 2D first?, my OCD is tingling

![imagen](https://user-images.githubusercontent.com/19991680/51632285-3430c680-1f4f-11e9-8670-374984644ccf.png)

Reply:
Ok, no problem